public/app/features/variables/editor/VariableEditorContainer.tsx

```diff
@@ -48,7 +48,7 @@ class VariableEditorContainerUnconnected extends PureComponent<Props> {
     this.props.switchToEditMode(identifier);
   };
 
-  onNewVariable = (event: MouseEvent<HTMLAnchorElement>) => {
+  onNewVariable = (event: MouseEvent<Element>) => {
```

Comment:
```suggestion
  onNewVariable = (event: MouseEvent) => {
```
`Element` is a default type on the MouseEvent, so it can be omitted here.

Reply:
Fixed!